### PR TITLE
Fix: cleanup ResourceRegistry controller logs

### PR DESCRIPTION
**Pull request by xigang**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```
NONE
```

### CODE REVIEW

File: pkg/search/controller.go [hunk 1]
```diff
@@ -410,7 +410,7 @@ func (c *Controller) deleteCluster(obj interface{}) {
 	c.queue.Add(cluster.GetName())
 }
 
-// getClusterAndResource returns the cluster and resources from the resourceRegistry object
+// getClusters return the cluster from the resourceRegistry object
```
Comment (XiShanYongYe-Chang) [inline pkg/search/controller.go:413]: return -> returns

File: pkg/search/controller.go [hunk 2]
```diff
@@ -426,7 +426,7 @@ func (c *Controller) getClusters(affinity policyv1alpha1.ClusterAffinity) []stri
 	return clusters
 }
 
-// getClusterAndResource returns the cluster and resources from the resourceRegistry object
+// getResources return the resources from the resourceRegistry object
```
Comment (XiShanYongYe-Chang) [inline pkg/search/controller.go:429]: return -> returns

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Thanks for you cleanup

File: pkg/search/controller.go (hunk 2 above)
Comment (xigang) [reply pkg/search/controller.go:429]: 👌

File: pkg/search/controller.go (hunk 1 above)
Comment (xigang) [reply pkg/search/controller.go]: @XiShanYongYe-Chang The code has been modified.

**Review by XiShanYongYe-Chang — COMMENTED** [review]
/lgtm

**Review by RainbowMango — APPROVED** [review]
/approve
Thanks.

**Pull request merged**